iina/MainWindowController.swift

```diff
@@ -1051,6 +1050,10 @@ class MainWindowController: PlayerWindowController {
       guard !isMouseEvent(event, inAnyOf: [currentControlBar]) else { return }
     }
 
+    if isMouseEvent(event, inAnyOf: [sideBarView, titleBarView, subPopoverView]) && !(seekOverride || volumeOverride) {
+      return
+    }
+
```

Comment:
Why switch to a complex `if` over a `guard`? Can't this just be?:
```swift
    guard !isMouseEvent(event, inAnyOf: [sideBarView, titleBarView, subPopoverView])
            || seekOverride || volumeOverride else { return }
```

Reply:
You are absolutely correct. Somehow I incorrectly thought converting to `if` would make it simpler.